tests/pr_test.go

```diff
@@ -63,30 +88,6 @@ func TestRunComplete(t *testing.T) {
 			"admin_pass": randomPass,
 		},
 	})
-	options.SkipTestTearDown = true
-	output, err := options.RunTestConsistency()
-	assert.Nil(t, err, "This should not have errored")
-	assert.NotNil(t, output, "Expected some output")
-	outputs := terraform.OutputAll(options.Testing, options.TerraformOptions)
-	expectedOutputs := []string{"port", "hostname"}
-	_, outputErr := testhelper.ValidateTerraformOutputs(outputs, expectedOutputs...)
-	assert.NoErrorf(t, outputErr, "Some outputs not found or nil")
-	options.TestTearDown()
-}
```

Comment:
Can you please add back the extra test logic here to check that port and hostname have values